src/gui/KM_InterfaceGame.pas

```diff
@@ -149,6 +158,12 @@ constructor TKMUserInterfaceGame.Create(aRender: TRender);
 
   fMinimap := TKMMinimap.Create(False, False);
   fViewport := TKMViewport.Create(aRender.ScreenX, aRender.ScreenY);
+
+  fDragScrolling := False;
+  fDragScrollingCursorPos.X := 0;
+  fDragScrollingCursorPos.Y := 0;
+  fDragScrollingViewportPos := ZERO_POINT_F;
```

Comment:
You know that in constructors all fields are initialized with "zeroes" right?

Adding a ZERO_POINT_F constant just for this is a bit unjustified.